doc/neps/nep-0049.rst

```diff
@@ -118,20 +118,20 @@ NumPy C-API functions
 
         typedef struct {
             char name[128];  /* multiple of 64 to keep the struct aligned */
-            PyDataMem_AllocFunc *alloc;
-            PyDataMem_ZeroedAllocFunc *zeroed_alloc;
-            PyDataMem_FreeFunc *free;
-            PyDataMem_ReallocFunc *realloc;
+            PyDataMemAllocator allocator; 
         } PyDataMem_Handler;
 
-    where the function's signatures are
+    where the allocator structure is
 
     .. code-block:: c
-
```

Comment:
The `PyDataMemAllocator` structure is not [rendered](https://20781-908607-gh.circle-artifacts.com/0/doc/neps/_build/html/nep-0049.html#c.PyDataMem_Handler) at all. It might just need that empty line.

Reply:
added a blank line